bblayers_imx6ul-des0258-mbl.conf

```diff
@@ -0,0 +1,20 @@
+# Based on: conf/bblayers.conf
+# In open-source project: https://github.com/96boards/oe-rpb-manifest
+#
+# Original file: Copyright (c) 2013 Khem Raj
+# Modifications: Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+
+# SPDX-License-Identifier: MIT
+
+# LAYER_CONF_VERSION is increased each time build/conf/bblayers.conf
+# changes incompatibly
+
+# These layers hold machine specific content, aka Board Support Packages
+BSPLAYERS += " \
+  ${OEROOT}/layers/meta-mbl/meta-freescale-mbl \
+  ${OEROOT}/layers/meta-freescale \
+  ${OEROOT}/layers/meta-mbl/meta-freescale-3rdparty-mbl \
+  ${OEROOT}/layers/meta-freescale-3rdparty \
+  ${OEROOT}/layers/meta-mbl/meta-fsl-bsp-release-mbl/imx/meta-bsp \
```

Comment:
`meta-fsl-bsp-release-mbl` and `meta-fsl-bsp-release` layers are not needed. This platform doesn't have the qca9377 wireless module.